### PR TITLE
adding pyhams to conda-forge

### PR DESCRIPTION
**Pull request by gbarter**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by gbarter** [comment]
@conda-forge/help-python-c

**Review by carterbox — CHANGES_REQUESTED** [review]

**Review by carterbox — CHANGES_REQUESTED** [review]
You are installing two copies of your library. One in site-packages/pyhams and another in site-packages/meson_build/pyhams. There should only be one copy.

**Comment by gbarter** [comment]
> You are installing two copies of your library. One in site-packages/pyhams and another in site-packages/meson_build/pyhams. There should only be one copy.

@carterbox - Very much appreciate the thorough review and cleanup.  I have adjusted the build backend settings so that there is only one copy of the compiled libraries.

**Comment by gbarter** [comment]
@mattEhall @danzalkind @lucas-carmo : Please comment here that you are willing to help maintain this conda package with me

**Comment by dzalkind** [comment]
Happy to help, but @dzalkind is the right guy for the job :-)

**Comment by gbarter** [comment]
> Happy to help, but @dzalkind is the right guy for the job :-)

Ah, my mistake.  I corrected the `meta.yaml` maintainers list.  Yes, @dzalkind please chime in that you can help here.

**Comment by dzalkind** [comment]
Willing

**Comment by mattEhall** [comment]
> @mattEhall @danzalkind @lucas-carmo : Please comment here that you are willing to help maintain this conda package with me

Happy to help

**Comment by gbarter** [comment]
@conda-forge/help-c-cpp @carterbox this is ready for review.  Please remove the `Awaiting author contributions` label.

File: recipes/pyhams/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+source:
+  url: https://github.com/WISDEM/pyHAMS/archive/v{{ version }}.tar.gz
+  sha256: 7d72a30cfb13be162c5aa2cea0e053ea82f5b0f638242f66452dc0dab454678e
+  # Download the tar-ball and do: openssl sha256 file.tar.gz to update
+
+build:
+  number: 0
+  skip: true  # [py<39]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```
Comment (carterbox) [inline recipes/pyhams/meta.yaml:15]:
```suggestion
  script:
    - {{ PYTHON }} -m pip install . --no-deps -vv
    - rm -rf {{ SP_DIR }}/meson_build       # [unix]
    - rmdir /s /y {{ SP_DIR }}/meson_build  # [win]
```
You're still installing some extra stuff into site packages.

**Review by carterbox — CHANGES_REQUESTED** [review]

File: recipes/pyhams/meta.yaml (hunk 1 above)
Comment (gbarter) [reply recipes/pyhams/meta.yaml]: That is a great suggestion in the script!

**Comment by gbarter** [comment]
@conda-forge/help-c-cpp @carterbox Thank you for the responsiveness and great suggestion in the recipe.  Please remove the `Awaiting author contribution` label.  Ready for another look.

**Pull request merged**